github/golang/stages/10-code-check/golangci-lint/action.yaml

```diff
@@ -0,0 +1,11 @@
+on:
+  workflow_call:
+
+runs:
+  using: 'composite'
+  steps:
+    - uses: 'actions/checkout@v3'
+    - uses: 'rios0rios0/pipelines/github/golang/abstracts/scripts-repo@feat/#19'
+    - run: $SCRIPTS_DIR/global/scripts/golangci-lint/run.sh
+      shell: 'bash'
+      continue-on-error: true
```

Comment:
@k4yt3x I was wondering if we have a yellow flag for the jobs that are failing but they are allowed to fail.

Reply:
As workaround, I was thinking in creating an output to set a yellow flag using this example below as inspiration:
![image](https://github.com/rios0rios0/pipelines/assets/32651906/827a4469-a78d-453f-919f-e4e4afa12ed3)